hq/app/logic/SnykDisplay.scala

```diff
@@ -0,0 +1,52 @@
+package logic
+
+import play.api.libs.json._
+import play.api.libs.json.Reads._
+import utils.attempt.{Attempt, FailedAttempt, Failure}
+
+import scala.concurrent.ExecutionContext
+
+import model._
+
+object SnykDisplay extends Serializers {
+
+  def getOrganisationId(s: String, organisation: Organisation) = {
```

Comment:
It is super helpful to have return type annotations on non-trivial functions (we don't always have IntelliJ to help e.g. in Github)

Reply:
Done.